backend/capellacollab/settings/modelsources/git/routes.py

```diff
@@ -28,9 +38,9 @@ def list_git_instances(
     response_model=models.GitInstance,
     dependencies=[
         fastapi.Depends(
-            auth_injectables.RoleVerification(
-                required_role=users_models.Role.ADMIN
-            )
+            permissions_injectables.PermissionValidation(
+                required_scope=permissions_models.GlobalScopes()
+            ),
```

Comment:
This endpoint used to only be accessible to admins, now it's accessible to all logged in users. Is this intentional?

Reply:
Yes. The endpoint to get all instances was already open, so it doesn't make sense to restrict the endpoint that returns a instance by ID.